pkg/orm/orm.go

```diff
@@ -54,12 +58,26 @@ func (c *OrmConfig) setDefaults(
 	}
 }
 
+// Orm - main structure for orm object
 type Orm struct {
 	*gorm.DB
+	config *OrmConfig
 }
 
+// Migration represents a database migration (a modification to be made on the database).
+type Migration struct {
+	// ID is the migration identifier. Usually a timestamp like "201601021504".
+	ID string
+	// Migrate is a function that will br executed while running this migration.
+	Migrate gormigrate.MigrateFunc
+	// Rollback will be executed on rollback. Can be nil.
+	Rollback gormigrate.RollbackFunc
+}
+
+// NewMySqlOrm - creates a new Orm object with MySQL connection
+// TODO: rename to NewMySQLOrm
```

Comment:
Why not right away? 😁

Reply:
Because it may be used in other projects... I decided not to risk...